### PR TITLE
Supporting retrieving a JSON Array from a POST request

### PR DESCRIPTION
**Pull request by Coeur**

### CODE REVIEW

**Comment by jdnichollsc** [comment]
Thanks mate, I'm going to publish this change in the next release! 👍

File: src/Proyecto26.RestClient/RestClientPromise.cs [hunk 1]
```diff
@@ -173,6 +173,33 @@ public static IPromise<T> Post<T>(RequestHelper options)
...
+        /// </summary>
+        /// <returns>Returns a promise for an array of values.</returns>
+        /// <param name="url">A string containing the URL to which the request is sent.</param>
+        /// <param name="body">A plain object that is sent to the server with the request.</param>
+        /// <typeparam name="T">The element type of the array.</typeparam>
+        public static IPromise<T[]> PostArray<T>(string url, object body)
+        {
+            var promise = new Promise<T[]>();
+            PostArray<T>(new RequestHelper { Uri = url, Body = body }, promise.Promisify);
+            return promise;
```
Comment (Coeur) [inline src/Proyecto26.RestClient/RestClientPromise.cs]: @jdnichollsc we could actually replace the three lines of those with a simple `return PostArray(new RequestHelper { Uri = url, Body = body })`.

Comment (jdnichollsc) [reply src/Proyecto26.RestClient/RestClientPromise.cs]: Let me see

Comment (jdnichollsc) [reply src/Proyecto26.RestClient/RestClientPromise.cs]: Fixed, thanks! https://github.com/proyecto26/RestClient/blob/master/src/Proyecto26.RestClient/RestClientPromise.cs#L183

**Pull request merged**